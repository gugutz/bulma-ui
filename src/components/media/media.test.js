import React from 'react'
import renderer from 'react-test-renderer'
import Media from './media'

describe('<Media /> component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Media>
          <Media.Left>
            <div>something</div>
          </Media.Left>
          <Media.Content>
            <div>something</div>
          </Media.Content>
          <Media.Right>
            <div>something</div>
          </Media.Right>
        </Media>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
