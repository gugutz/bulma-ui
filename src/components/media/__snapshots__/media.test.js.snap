// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Media /> component should render correctly 1`] = `
<div
  className="media"
>
  <div
    className="media-left"
  >
    <div>
      something
    </div>
  </div>
  <div
    className="media-content"
  >
    <div>
      something
    </div>
  </div>
  <div
    className="media-right"
  >
    <div>
      something
    </div>
  </div>
</div>
`;
